lms/djangoapps/bulk_email/tasks.py

```diff
@@ -418,7 +418,22 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         'failed' count above.
     """
```

Comment:
@mikigoyal , yeah, I think there's enough information in each log that that'll be pretty straightforward